lib/DocumentCard/DocumentCard.js

```diff
@@ -59,13 +63,26 @@ export default class DocumentCard extends React.Component {
         <Headline data-test-doc-name={name} margin="none">
           {name}
         </Headline>
-        {note ?
-          <KeyValue label={<FormattedMessage id="stripes-erm-components.doc.note" />}>
-            <div data-test-doc-note>{note}</div>
-          </KeyValue>
-          :
-          null
-        }
+        <Row>
+          <Col xs={8}>
+            {note ?
+              <KeyValue label={<FormattedMessage id="stripes-erm-components.doc.note" />}>
+                <div data-test-doc-note>{note}</div>
+              </KeyValue>
+              :
+              null
+            }
+          </Col>
+          <Col>
```

Comment:
This should have an `xs={category ? 4 : 0}` for completeness. Generally, always have a size when using `Col`.